modules/manage/pages/manage-security/configure-ldap.adoc

```diff
@@ -438,18 +475,18 @@ To configure an LDAP server to be used as a point of authentication for Couchbas
 --password password \
 --hosts 172.23.124.25 --port 389 \
 --encryption startTLS \
---ca-cert '/path/to/cert' \
---query-dn 'cn=admin,dc=example,dc=com' \
---query-pass 'password' \
+--ldap-cacert '/path/to/cert' \
+--user-dn-query 'cn=admin,dc=example,dc=com' \
```

Comment:
This is actually --bind-dn

Reply:
I've changed --user-dn-query to --bind-dn, and have kept the value the same.